web.py

```diff
@@ -14,3 +16,26 @@ def home():
 def sim(index):
     graph_json = SIMULATION[index]
     return graph_json
+
+
+app.config["UPLOADS"] = "/workspaces/MARLon/uploads"
+
+@app.route('/upload-file', methods=['POST'])
+def upload_file():
+    
+    if request.files['attackerFile']:
+        afile = request.files['attackerFile']
+    else:
+        print("Missing Attacker File")
+        return render_template('index.html', max_sim=len(SIMULATION) - 1)
+    if request.files['defenderFile']:
+        dfile = request.files['defenderFile']
+    else:
+        print("Missing Defender File")
+        return render_template('index.html', max_sim=len(SIMULATION) - 1)
```

Comment:
You should be allowed to upload only a single agent (with the default being a random agent). Especially important right now because we can only train attackers.